Sources/SwifQL/SwifQLSelectBuilder.swift

```diff
@@ -8,18 +8,33 @@
 import Foundation
 
 public class SwifQLSelectBuilder {
-    var select: [SwifQLable] = []
-    var froms: [SwifQLable] = []
-    var joins: [SwifQLJoinBuilder] = []
-    var wheres: [SwifQLable] = []
-    var groupBy: [SwifQLable] = []
-    var havings: [SwifQLable] = []
-    var orderBy: [OrderByItem] = []
-    var offset: Int?
-    var limit: Int?
+    private var select: [SwifQLable] = []
```

Comment:
What's the point for private variables here? 🙂 They're internal by themselves and not available outside the lib, but I'm aware that maybe in the future I'd write some extensions for select builder and private variable will not allow me to do that.

Reply:
So yeah I think that there is no need to mark these properties as `private` for now. Please remove `private` 🙂

Reply:
done